medicine_search_bot.py

```diff
@@ -0,0 +1,1267 @@
+                '\n\n☝️️ Зверніть увагу, ви можете надсилати *одразу декілька фотографій*\.'
+                '\n\n❗️ Переконайтесь, що фотографія *не розмита*, а штрихкод розташований '
+                '*вертикально* або *горизонтально*\.'
+                '\nЦе мінімізує кількість помилок та дозволить боту працювати коректно\.'
+                '\n\n🔍 *Скористатися розширеним пошуком* можна за допомогою команди */search* '
+                '\nЦя функція дозволяє здійснювати пошук за *назвою*, *діючою речовиною*, а '
+                'також *штрих\-кодом* медикаменту\.'
+                '\n\n ↩️ Відмінити *будь\-яку дію* можна командою */cancel*'
+                '\n\n ⚙️ За допомогою команди */settings* можна налаштувати функцію *пошуку '
+                'медикаменту у Google*'
+                '\n\n📩 Надіслати нам відгук можна обравши опцію "*Надіслати відгук*" із '
+                'головного меню, або скориставшись командою */feedback*'
+                '\n\n 💬 Ви можете викликати це повідомлення у *будь\-який момент*, '
+                'надіславши команду */help*',
+        parse_mode='MarkdownV2',
+        reply_markup=ReplyKeyboardMarkup(
+            reply_keyboard,
+            one_time_keyboard=True,
+            resize_keyboard=True,
+            input_field_placeholder='Оберіть опцію',
+        ),
+    )
+
+
+@under_maintenance
+def goto_scan(update: Update, context: CallbackContext) -> None:
+    """
+    The goto_scan function is called in order to call scan_handler.
+    It will remove any reply_keyboard that is present and
+    tells the user that they can scan their documents.
+
+    :param update:Update: Access the message that was sent by the user
+    :param context:CallbackContext: Keep the user's data
+    :return: None
+    """
+    user = update.message.from_user
+    logger.info("%s: %s", user.first_name, update.message.text)
+
+    reply_keyboard = MAIN_REPLY_KEYBOARD
+
+    if update.message.text != 'Ще раз':
+        update.message.reply_text(
+            text='Гаразд',
+            reply_markup=ReplyKeyboardMarkup(
+                reply_keyboard, one_time_keyboard=True,
+                resize_keyboard=True,
+                input_field_placeholder='Надішліть фото',
+            ),
+        )
+        return
+    return scan_handler(update=update, context=context)
+
+
+def get_db_query_result(barcode) -> bool or None:
+    """
+    The get_db_query_result function takes a barcode as an argument and returns the result of a MongoDB query.
+    If no results are found, it returns None.
+
+    :param barcode: Check if the barcode exists in the database
+    :return: A dictionary with the product information if the barcode exists in the database
+    """
+    try:
+        logger.info("Database quired. Checking availability")
+        query_result = collection.find_one({"code": barcode}, {"_id": 0})
+        if query_result is None:
+            return
+
+        return query_result
```

Comment:
```suggestion
        return query_result
```
If statement is redundant, since method `find_one` returns `None` if nothing's found - read [here](https://pymongo.readthedocs.io/en/stable/tutorial.html#getting-a-single-document-with-find-one)

Reply:
Accepted